projects/ghostscript/build.sh

```diff
@@ -52,6 +52,7 @@ make -j$(nproc) libgs
 $CXX $CXXFLAGS $CUPS_LDFLAGS -std=c++11 -I. \
     fuzz/gstoraster_fuzzer.cc \
     -o "$OUT/gstoraster_fuzzer" \
+    -Wl,-rpath=. \
```

Comment:
Does this work if I do this:
```
cd /some/path/not/containing/the/fuzzer
/path/to/the/fuzzer
```
?

Reply:
I'm not a linking expert, but I think $ORIGIN might be what we need.

Reply:
No, I'm pretty sure that won't work.  I've changed it to $ORIGIN and I think it's correct now.